proto/p4/p4runtime.proto

```diff
@@ -154,8 +154,8 @@ message FieldMatch {
     int32 prefix_len = 2;  // in bits
   }
   message Range {
```

Comment:
Suggest a comment to clarify ... something like (taken from p4-16 spec):
A Range is logically a set that contains all values numerically between 'low' and 'high' inclusively.
P4_16 defines the two arguments ('low' and 'high') to be of type bit<W> or int<W>. Since W can be arbitrarily wide, this is modeled as protobuf type 'bytes'.